### DISCUSSION TITLE
Should I use Link component with external links ?

### DISCUSSION THREAD

**Discussion by RigottiG** [vercel/next.js#37597] [Help] [answered]
I'm making a list of dynamic links where there'll be external and internal links. can the Link component be used in external links without problem?

**Comment by icyJoseph** [comment, marked as answer]
Yes. Be careful with `schema` links, which look like this: `<a href="//example.com">Scheme-relative URL</a>` (of course using `Link`), which don't really work out as is. Otherwise it is alright. 

The only reason to use Link is to let the router know that the user wants to navigate, and that the SPA experience should happen, but if the links is absolute, and external (it could still point to the same domain), then it is a NOOP.

**Reply by SukkaW** [reply]
> Yes. Be careful with `schema` links, which look like this: `<a href="//example.com">Scheme-relative URL</a>` (of course using `Link`), which don't really work out as is. Otherwise it is alright.

Next.js has fixed the issue already! https://github.com/vercel/next.js/pull/36823
